diff -ruN kos-ports/scripts/clean.mk kos-ports-offline/scripts/clean.mk
--- kos-ports/scripts/clean.mk	2020-01-18 12:44:11 +0000
+++ kos-ports-offline/scripts/clean.mk	2020-01-18 16:39:33 +0000
@@ -5,8 +5,9 @@
 #
 
 clean-dist:
-	@echo "Cleaning up dist files ..."
-	-@rm -rf dist
+	@echo "Dist files weren't cleaned up as your are using an offline trunk..."
+#	@echo "Cleaning up dist files ..."
+#	-@rm -rf dist
 
 clean-build:
 	@echo "Cleaning up build directories ..."
